src/kakarot/account.cairo

```diff
@@ -366,6 +366,68 @@ namespace Account {
         let starknet_account_exists = is_not_zero(registered_starknet_account);
         return starknet_account_exists;
     }
+
+    // @notice Initializes a dictionary of valid jump destinations in EVM bytecode.
+    // @param i The current index in the bytecode.
+    // @param bytecode_len The length of the bytecode.
+    // @param bytecode The EVM bytecode to analyze.
+    // @param valid_jumpdests A dictionary to be filled with valid jump destinations.
+    // @return valid_jumpdests The updated dictionary with valid jump destinations.
+    //
+    // @dev This function iterates over the bytecode from the current index 'i'.
+    // If the opcode at the current index is between 0x5f and 0x7f (PUSHN opcodes) (inclusive),
+    // it skips the next 'n_args' opcodes, where 'n_args' is the opcode minus 0x5f.
+    // If the opcode is 0x5b (JUMPDEST), it marks the current index as a valid jump destination.
+    // The function then increments the index and recursively calls itself until it has processed the entire bytecode.
+    func init_valid_jumpdests{range_check_ptr}(bytecode_len: felt, bytecode: felt*) -> (
```

Comment:
could be `get_valid_jumpdest`, or even `get_jumpdests`, at least the `init` sounds like we're mutating something, which is not the case anymore